apps/site/src/lib/components/ValidatingForm/ValidatingForm.tsx

```diff
@@ -11,12 +12,15 @@ interface FormProps {
 
 function ValidatingForm(props: PropsWithChildren<FormProps>) {
 	const [validated, setValidated] = useState<boolean>(false);
+	const [isDisabled, setIsDisabled] = useState<boolean>(false);
```

Comment:
Suggestion/nitpick: I think `submitting, setSubmitting` would be more descriptive than `isDisabled`.